e2e/cli_test.go

```diff
@@ -0,0 +1,81 @@
+package e2e_test
+
+import (
+	"os"
+	"os/exec"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gbytes"
+	"github.com/onsi/gomega/gexec"
+)
+
+var _ = Describe("CLI", func() {
+	act := func(arg ...string) (session *gexec.Session, err error) {
+		cmd := exec.Command(cliPath, arg...)
+		session, err = gexec.Start(cmd, GinkgoWriter, GinkgoWriter)
+		return
+	}
+
+	Describe("help", func() {
+		It("should show the help for server", func() {
+			session, err := act("help")
+			Expect(err).ToNot(HaveOccurred())
+			Eventually(session.Out).Should(gbytes.Say(`Usage:`))
+			Eventually(session.Out).Should(gbytes.Say(`-h, --help`))
+			Eventually(session.Out).Should(gbytes.Say(`-c, --kubeconfig string`))
```

Comment:
Is there any value in testing the output for all of the currently available options and commands?
I think it's good to test that the command works in general, but I can't really think of a case where testing all of its output would catch any real issue.

Reply:
I want to make sure that all options are there and no additional options have been added without looking at the tests. I'll adapt this test, so it's more like the 'shows all available commands'.